.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,21 @@
+name: Pull Request Tests
+
+on:
+  pull_request:
+    branches:
+      - "**"
+
+concurrency: 
+  group: ${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  unit-integration-tests:
+    strategy:
+      matrix:
+        go-version: ["1.17"]
```

Comment:
Ah, didn't notice that this was Go 1.19 before. Agreed that we should run the oldest supported version 👍 Good catch.

Reply:
Thanks @ajgajg1134